src/React/DOM/Props.js

```diff
@@ -19,7 +16,7 @@ function unsafeUnfoldProps(key) {
     props[key] = result;
 
     for (var subprop in value) {
-      if (value.hasOwnProperty(subprop)) {
+      if (Object.prototype.hasOwnProperty.call(value, subprop)) {
```

Comment:
The changes in this file have to do with eslint's rule about using prototype builtins:

https://eslint.org/docs/rules/no-prototype-builtins